eng/Versions.props

```diff
@@ -116,7 +116,7 @@
     <MicrosoftServiceHubClientVersion>2.8.10</MicrosoftServiceHubClientVersion>
     <MicrosoftServiceHubFrameworkVersion>2.8.10</MicrosoftServiceHubFrameworkVersion>
     <MicrosoftVisualBasicVersion>10.1.0</MicrosoftVisualBasicVersion>
-    <MicrosoftVisualStudioCacheVersion>16.10.40-alpha</MicrosoftVisualStudioCacheVersion>
+    <MicrosoftVisualStudioCacheVersion>17.0.13-alpha</MicrosoftVisualStudioCacheVersion>
```

Comment:
might need to go into main-vs-deps unless we're shipping this in VS, I don't think we've merged dev17 back into main yet (though hopefully soon)

Reply:
main-vs-deps already has this.  this is to unblock main through from teh test failures we're seeing there.  do you know when we're going to be on dev17 for main?  @jinujoseph ?

Reply:
I believe we're waiting on preview2 images? @allisonchou was working on it actively if I remember right

Reply:
@dibarbet can this be merged in now?

Reply:
Yep I think so!